packages/knip/src/plugins/sst/index.ts

```diff
@@ -0,0 +1,24 @@
+import type { IsPluginEnabled, Plugin } from "../../types/config.js";
+import { hasDependency } from "../../util/plugin.js";
+
+// link to sst docs
+
+const title = "sst";
+
+const enablers = ["sst"];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) =>
+  hasDependency(dependencies, enablers);
+
+const entry = [
+  "sst.config.{js,cjs,mjs,ts}",
+  "{handlers,lambdas}/*.{js,cjs,mjs,ts}",
+  "src/{handlers,lambdas}/*.{js,cjs,mjs,ts}",
```

Comment:
Do we have any docs/resources that indicate these could be considered defaults locations?